CONTRIBUTING.md

```diff
@@ -93,15 +93,15 @@ docker run --rm --network determined --name determined-db \
   -e POSTGRES_PASSWORD=my-postgres-password \
   postgres:10
 
-# Start the master.
-make -C master install-native
-determined-master \
+# Build and start the master.
+make -C master build
+./master/build/determined-master \
```

Comment:
blocking: have you tried running through this locally?

Reply:
Yes.

Reply:
I tried running this and I got the following:
```
ERRO[2020-05-12T14:46:19-07:00] open /Users/yoni/Development/src/github.com/determined-ai/determined/build/share/determined/master/static/migrations: no such file or directory
```

This is on a clean install on the tip of master. Have you rebased off of master?

Reply:
Hmm, weird. I've rebased off of master and done a `make clean` but didn't completely `rm -rf` and restart -- maybe my local build env has some stuff hanging around from the old build system (e.g., `build/share/determined/master/static/migrations` exists)?

Can you suggest a revised command for running the master?

Reply:
Maybe pointing to the tools dir and using that instead?

Reply:
Ah, cool -- I missed the `tools/` stuff, that seems much better.